Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -766,7 +769,7 @@ YUI.add('ez-platformuiapp', function (Y) {
                     path: "/admin/:uri",
                     sideViews: {'navigationHub': true, 'discoveryBar': false},
                     service: Y.eZ.ServerSideViewService,
-                    view: "serverSideView",
+                    view: "contentTypeServerSideView",
```

Comment:
related to https://github.com/ezsystems/PlatformUIBundle/pull/261/files#r33333706 you should add a new route that matches only the content type edition